test/checks/color/color-contrast.js

```diff
@@ -103,6 +103,34 @@ describe('color-contrast', function () {
 		assert.deepEqual(checkContext._relatedNodes, [target]);
 	});
 
+
+	it('should ignore position:fixed elements above the target', function () {
```

Comment:
@WilcoFiers can you see whether there are any other test cases we should add to this?